aten/src/ATen/native/ao_sparse/quantized/cpu/qlinear_deserialize.cpp

```diff
@@ -31,6 +31,7 @@ std::vector<VEC_DTYPE> unwrap_vector(at::Tensor tensor) {
   return vec;
 }
 
+#ifdef USE_PYTORCH_QNNPACK
```

Comment:
This functions, unpack_bcsr and pack_bcsr, are only used with fbgemm so I am not sure why it needs these defs at all. From the build failure you got it seem to suggest ao::sparse::BCSR not defined, maybe because you are also building with USE_FBGEMM=0? In that case ifdef should be for USE_FBGEMM. cc: @salilsdesai